azure-client-runtime/src/main/java/com/microsoft/azure/AzureEnvironment.java

```diff
@@ -202,6 +209,11 @@ public String graphEndpoint() {
     }
 
     /**
+     * @return the Data Lake resource ID.
+     */
+    public String dataLakeEndpointResourceId() { return endpoints.get("dataLakeEndpointResourceId"); }
```

Comment:
[ERROR] src/main/java/com/microsoft/azure/AzureEnvironment.java:[214,48] (blocks) LeftCurly: '{' at column 48 should have line break after.